### DISCUSSION TITLE
How to filter out new followers from my feed?

### DISCUSSION THREAD

**Discussion by twpayne** [community/community#77258] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I've got several tens of new followers in the past 24 hours. The vast majority of these are obviously bot accounts. How can I filter new followers out of my https://github.com/dashboard-feed so I can focus on what is important?

**Comment by MadScie254** [comment]
To filter out new followers on GitHub and identify bot accounts, you can consider the following steps:

1. Evaluate follower activity: Look for suspicious patterns in the activity of your followers. Check if they have a history of following and unfollowing quickly or if they engage in spammy behaviour.

2. Use automation tools: There are several third-party tools available that can help you identify and filter out bot accounts. These tools analyze the behaviour and characteristics of the followers and provide insights into their authenticity.

3. Implement limitations: Set a limit on the number of users someone can follow each day. This can discourage users from employing automated tools to gain followers quickly.

4. Report and ban suspicious accounts: If you come across obvious bot accounts, report them to GitHub support. Additionally, consider banning users who follow a large number of people in a short period.

5. Customize notification settings: GitHub allows you to customize your notification settings. You can choose to receive notifications only for actions that are relevant to you, such as pull requests or issue comments. This can help reduce the noise caused by spammy followers.

**Reply by twpayne** [reply]
ChatGPT, is that you?

> 5\. Customize notification settings: GitHub allows you to customize your notification settings. You can choose to receive notifications only for actions that are relevant to you, such as pull requests or issue comments. This can help reduce the noise caused by spammy followers.

I'm asking how to do this.

**Reply by MadScie254** [reply]
As an AI model I would love to say that is the correct answer😂😂😂😂

**Reply by MadScie254** [reply]
Ok!.. on a serious note, Install a browser extension like FakerFinder that uses heuristics to identify bot/fake accounts and allow you to filter or block them.
